Add important info about http_host and file formatting to s3 example

avrittrohwer opened the pull request:
Add important info about http_host and file formatting to s3 example

Will save future folks from having to debug the same thing I did.

axboe commented:
Commit messages (and signed-off-by) need to go into the commit itself, not be part of the GH PR message.

avrittrohwer commented:
> Commit messages (and signed-off-by) need to go into the commit itself, not be part of the GH PR message.

Fixed

axboe commented:
It's not very descriptive. What does "very important" mean?

```
examples/s3: add info about http_host and file formatting

This caused me some headache, let's add some details on how fio expects the
http_host and file to be formatted.

Signed-off-by: Avritt Rohwer <avritt@google.com>
```

I think would be much better.

avrittrohwer commented:
> It's not very descriptive. What does "very important" mean?
> 
> ```
> examples/s3: add info about http_host and file formatting
> 
> This caused me some headache, let's add some details on how fio expects the
> http_host and file to be formatted.
> 
> Signed-off-by: Avritt Rohwer <avritt@google.com>
> ```
> 
> I think would be much better.

Sure.  Also added some more info on required steps for enabling the http engine while building and added the leading / info to the docs too

axboe commented:
Thanks! You still missed the ```<>``` around your email, but let's just get it in.

sitsofe commented:
@avrittrohwer would you be able to update the man page too? See https://github.com/axboe/fio/blob/f4e54d3fa7bf94a17648a4439f06ca7dab4291c0/fio.1#L591-L593 ...

avrittrohwer commented:
> @avrittrohwer would you be able to update the man page too? See
> 
> https://github.com/axboe/fio/blob/f4e54d3fa7bf94a17648a4439f06ca7dab4291c0/fio.1#L591-L593
> 
> ...

From https://github.com/axboe/fio/blob/f4e54d3fa7bf94a17648a4439f06ca7dab4291c0/README.rst#documentation it seemed like the man page is generated, is that step required manually by editors or taken care of automatically via CI?
